VictoriaAlbert/VictoriaAlbert/SearchResultsTableViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  SearchResultsTableViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Jermaine Kelly on 11/10/16.
+//  Copyright © 2016 Jermaine Kelly. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewController: UITableViewController, UISearchBarDelegate {
+    
+    
+    var results: [Record] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        makeSearchBar()
+        getResults()
+        
+    }
+    
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return results.count
+    }
+    
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "resultsCell", for: indexPath) as! ResultTableViewCell
+        
+        let record = results[indexPath.row]
+        cell.nameLabel.text = "\(record.name), \(record.date), \(record.place)"
+        cell.titleLabel.text = record.title
+        
+        var imageIdForUrl = ""
+        
+        for (index,character) in record.imageID.characters.enumerated(){
```

Comment:
This imageID parsing code is repeated in the detail view controller. Consider refactoring it to one place, probably the model.